app/channels/editor_channel.rb

```diff
@@ -0,0 +1,59 @@
+# The Editor Channel is opened when a user navigates to the `Edit` page of an
+# Evidence, Issue, Note, or Card. The channel is closed when the user navigates
+# away from the `edit` page. When open the channel receives updates from the
+# editor and saves them as new revisions of the resource.
+class EditorChannel < ApplicationCable::Channel
+  include ProjectScopedChannels
+  include PaperTrailActivity
+
+  attr_accessor :resource
+
+  AUTOSAVE_EVENT = 'auto-save'.freeze
+
+  def subscribed
+    reject and return unless find_resource(params)
+
+    stream_for [current_user, current_project, resource]
+  end
+
+  def save(params)
+    resource.paper_trail_event = AUTOSAVE_EVENT
+
+    if resource.update_attributes parsed_params(params['data'])
+      track_activity(resource, AUTOSAVE_EVENT)
```

Comment:
I don't think we need to track the activity for the auto-save. On the other hand, revisions are okay once we have a way to collapse the revisions.

One idea is to have an activity that announces that someone started editing a resource like: `admin@securityroots.com started editing Issue #1` instead. But, I don't think the current implementation of activity permits this, so I'm inclined to skip activities for now.

Reply:
Oh you're so right!

All the auto-save activity would just be straight noise. It's not useful to anyone. I'll pull this out asap.